java/tsfile/src/main/java/org/apache/tsfile/utils/ReadWriteIOUtils.java

```diff
@@ -95,6 +95,12 @@ public static boolean readBool(ByteBuffer buffer) {
     return a == 1;
   }
 
+  /** read a bool from byteBuffer. */
+  public static boolean readBoolean(ByteBuffer buffer) {
```

Comment:
why not using existing `readBool`?

Reply:
Even if you want to keep the code same for each type in serialize method for generated code, you still have another two ways to achieve this goal:
1. call readBool in readBoolean
2. add if-else and specify readBool for boolean type in ftl.